lib/aruba/api.rb

```diff
@@ -1129,7 +1131,7 @@ def unset_bundler_env_vars
     # @param [String] value
     #   The value of the environment variable. Needs to be a string.
     def set_env(key, value)
-      announcer.env(key, value)
+      announcer.announce(:environment, key, value)
```

Comment:
I changed the interface of the announcer to make it easier to extend. Now you just to need `announcer.active :channel1` to use `announcer.announce(:channel1, 'text')`. Maybe we could even make this API public. WDYT?